src/handlers/rfqm_handlers.ts

```diff
@@ -0,0 +1,63 @@
+// tslint:disable:max-file-line-count
+import { BigNumber } from '@0x/utils';
+import * as express from 'express';
+import * as HttpStatus from 'http-status-codes';
+
+import { CHAIN_ID } from '../config';
+import { schemas } from '../schemas';
+import { RfqmService } from '../services/rfqm_service';
+import { FetchIndicativeQuoteParams } from '../types';
+import { schemaUtils } from '../utils/schema_utils';
+import { findTokenAddressOrThrowApiError, isETHSymbolOrAddress } from '../utils/token_metadata_utils';
+
+export class RfqmHandlers {
+    constructor(public readonly _rfqmService: RfqmService) {}
+
+    public async getIndicativeQuoteAsync(req: express.Request, res: express.Response): Promise<void> {
+        const params = parseFetchIndicativeQuoteParams(req);
+        const indicativeQuote = await this._rfqmService.fetchIndicativeQuoteAsync(params);
+        res.status(HttpStatus.OK).send(indicativeQuote);
+    }
+}
+
+const parseFetchIndicativeQuoteParams = (req: express.Request): FetchIndicativeQuoteParams => {
```

Comment:
Things that I'd love to validate, that I think should be added to here:
- Make sure that the API key is RFQM-enabled
- Make sure the request is not for buy/sell ETH

Reply:
- Make sure `takerAddress` is not a smart contract

Reply:
- 👍 
- 👍 
- skip for now, but add a TODO

Reply:
Was about to add the TODO, but then realized that an increasing number of smart contract wallets might cause issues with this assumption. Could be a good parking lot item to discuss